core/modules/simpletest/tests/menu.test

```diff
@@ -1496,6 +1496,9 @@ class MenuTrailTestCase extends MenuWebTestCase {
     // the breadcrumbs and Management tree.
     state_set('menu_test_menu_tree_set_path', $test_menu_path);
     $this->assertBreadcrumb('admin/config/development/menu-trail', $override_breadcrumb, t('Menu trail - Case 2'), $override_tree);
+
+    // Load one more page to clear out any pending theme registry saves.
+    $this->backdropGet('admin/config/system');
```

Comment:
This is the weirdest failure I ran into with this PR. If there are pending saves to the theme registry, the `BackdropWebTestCase::setUp()` method tries to clear the theme registry but a new theme function tries to get written to cache while switching between config directories. This seems to be the only place in all of core where this particular problem occurred.

Reply:
I was able to remove this and replace it with a better solution for all tests in `BackdropWebTestCase::tearDown()`.